### ISSUE TITLE
download file from UV viewer does not work

### ISSUE THREAD

**Issue by hannahfrost** [samvera-labs/hyku#1223]

**Comment by jcoyne** [comment]
I suspect this can be fixed if it's prioritized.

**Comment by mjgiarlo** [comment]
Disable download button, and revisit after OR2017 (to re-enable the button and make it work)

**Comment by jcoyne** [comment]
The button is working for me now. However, there is a problem with "Current view"  See: 
https://github.com/UniversalViewer/universalviewer/issues/464

**Comment by hannahfrost** [comment]
@jcoyne I tested it just now on Hyku demo. Definitely an improvement, even if blocked by the UV bug. Unfortunately, when I try to download the other sizes, the image just opens in a new browser window; download is not initiated by the browser (both Safari 10.1.1 and Chrome 58.0.3). Also the file downloaded is named with the AWS string, not the original filename. 

Write these issues in a separate ticket and mark this ticket as blocked by the UV bug?

**Comment by jcoyne** [comment]
@hannahfrost Since that's in universal viewer, it has no knowledge of the repository. It's just downloading from the IIIF image api (it's the IIIF URL, not the aws URL by the way).  You'll see the same in other UV sites. For example: http://bluemountain.princeton.edu/exist/apps/bluemountain/issue.html?titleURN=bmtnaae&issueURN=bmtnaae_1918-12_01#?#viewer&c=0&m=0&s=0&cv=1&z=-3555.8478%2C0%2C11779.6956%2C6532

**Comment by hannahfrost** [comment]
@jcoyne Of course, that makes sense.
